### PR TITLE
cleanup tmp before build

### PR DESCRIPTION
**Pull request by RafLeszczynski**
## Links

* https://github.com/broccolijs/broccoli/issues/187

## Description

For some unknown reason `tmp/` directory is not always cleaned by `ember-cli`. Main 
had move than 4.5Gb and made my devbox run out of space. I'm adding temporary walk-around for this issues. Whe should revisit this case ones we upgrade to future versions of `ember-cli`.

## Reviewers
@Wikia/x-wing @dianafa @SebastianMarzjan @bkoval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "restart": "npm-run-all --parallel watch-auth watch-common watch-main watch-server start-on-change",
     "watch-auth": "gulp watch-auth",
     "watch-common": "gulp watch-common",
-    "watch-main": "cd front/main && ember build --watch",
+    "watch-main": "cd front/main && rm -rf temp && ember build --watch",
```
Comment (tomasznapieralski) [inline package.json]: how about emptying not removing dir?
`rm -rf temp/*`

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "restart": "npm-run-all --parallel watch-auth watch-common watch-main watch-server start-on-change",
     "watch-auth": "gulp watch-auth",
     "watch-common": "gulp watch-common",
-    "watch-main": "cd front/main && ember build --watch",
+    "watch-main": "cd front/main && rm -rf tmp && ember build --watch",
```
Comment (tomasznapieralski) [inline package.json]: how about emptying not removing dir?
`rm -rf tmp/*`

Comment (dianafa) [inline package.json]: +1

**Comment by rogatty** [comment]
LGTM

File: package.json (hunk 2 above)
Comment (RafLeszczynski) [inline package.json]: @tomasznapieralski @dianafa `tmp` directory is created by `ember-cli` on build so we can safely remove it

Comment (tomasznapieralski) [inline package.json]: :+1:

**Pull request merged**